sdks/python/apache_beam/runners/portability/portable_runner.py

```diff
@@ -74,6 +79,149 @@
 _LOGGER = logging.getLogger(__name__)
 
 
+class PortableJobServicePlan(object):
```

Comment:
Not sure about the name. Perhaps `JobServiceHandle`?

Reply:
yeah, I was unsure on that as well.  Open to input.

Reply:
I went with `JobServiceHandle`